### ISSUE TITLE
The meaning of "SPN" is nowhere explained

### ISSUE THREAD

**Issue by privacytoolsIO** [safing/web#245]
Hey, the word "SPN" is nowhere explained on the safing.io website. It's just explained in the GitHub browser title. Currently adding SPN to: https://www.privacytools.io/dvpn

**Comment by davegson** [comment]
thanks for the input and the listing Burung! 💙

----

Two thoughts on the listing:

- Cure 53 [audited the core encryption technology](https://cure53.de/pentest-report_safing-jess.pdf) of SPN in 2020. I'd be happy to get the "Audited" tag too
- also, was there a reason behind the ordering? I found it odd to see an early stage project listed first. And out of the listed projects [excluding Tor], I personally feel SPN is most mature and most private (being as unbiased as a co-founder can be haha).

Anyway, just wanted to share those 2c. Use as you see fit. Also, I added several tooltips on the [SPN page](https://safing.io/spn/) as well as on [comparison charts](https://safing.io/blog/2022/09/06/spn-vs-vpns/) - I feel this should clear things up for future visitors.

Thanks again for pointing this out!
